src/include/sof/lib/alloc.h

```diff
@@ -143,7 +148,7 @@ void rfree(void *ptr);
 #define rballoc(zone, caps, bytes)			\
 	({void *_ptr;					\
 	do {						\
-		_ptr = _balloc(zone, caps, bytes);	\
+		_ptr = _balloc(zone, caps, bytes, 0);	\
```

Comment:
Default alignment should be cache line size. It will be due to the buffer placement, but it's always better to show it explicitly.